health_check/contrib/redis/backends.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.conf import settings
+from redis import exceptions, from_url
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceUnavailable
+
+logger = logging.getLogger(__name__)
+
+
+class RedisHealthCheck(BaseHealthCheckBackend):
+    """Health check for Redis."""
+
+    def check_status(self):
+        """Check Redis service by opening and closing a connection."""
+        logger.debug("Checking for a redis_url in django settings...")
+
+        redis_url = getattr(settings, "REDIS_URL", None)
```

Comment:
Can you make that a class attribute? I for one, have multiple Redis servers running, so I will need to subclass this health check.

Reply:
Good idea! Added it